eslint/babel-eslint-parser/test/babel-eslint-parser.js

```diff
@@ -7,7 +7,10 @@ import { parseForESLint } from "../src";
 import assertImplementsAST from "./helpers/assert-implements-ast";
 
 const babelOptions = {
-  configFile: path.resolve(__dirname, "./fixtures/config/babel.config.js"),
+  configFile: path.resolve(
+    __dirname,
+    "../../babel-eslint-shared-fixtures/config/babel.config.js",
```

Comment:
You can add `@babel/eslint-shared-fixtures` as devDependency (even if that is private, but it should be okay to be dev deps) and use
```js
configFile: "@babel/eslint-shared-fixtures/config/babel.config.decorators-legacy.js"
```